worker/lease/bound.go

```diff
@@ -76,6 +76,12 @@ func (b *boundManager) Token(leaseName, holderName string) lease.Token {
 	}
 }
 
+// Pinned (lease.Pinner) returns applications and vested entities for
+// pinned leases in the bound namespace/model.
+func (b *boundManager) Pinned() map[string][]names.Tag {
+	return b.manager.pinned(b.namespace, b.modelUUID)
```

Comment:
I implemented this directly on manager rather than feeding another op into the worker loop. I _think_ this is fine because:
- The Raft FSM implementation has locking that prevents concurrent read/write access to pinned leases.
- Querying for pins has no behavioural affects, so strict synchronous execution in the loop is not necessary.